### ISSUE TITLE
Opening advertising subframes...

### ISSUE THREAD

**Issue by amenadiel** [callumlocke/json-formatter#151]
Dude, I love this extension. I've used it since forever.
However, since a few days ago, some sites made my browser to become "stuck". Meaning I couldn't click on other tabs, the page became unresponsive, the scrollbar went back and forth and it seemed like there was an overlay on top of it all.

I saw in task manager a lot of subframes of advertising networks or platform (openx, amazon, taobao, you name it). Double clicking on a subframe will take you to the opener, which in this case is your extension.

![json_formatter](https://user-images.githubusercontent.com/238439/68503983-da319100-0242-11ea-844c-56e79bcc889b.png)

I would bet you have little or no spare time so this extension doesn't get much maintenance, therefore it's a good target for extension-theft

**Comment by FahadYousafMahar** [comment]
@ffflabs Yeah I was also getting this and found out by stopping and restarting every extension. This extension is currently trying to inject JS from different .xyz and russian .ru domains. 
It tries to add video players and advertisement frames.
